src/contrib/cluster/Akka.Cluster.Sharding/ShardingMessages.cs

```diff
@@ -141,42 +142,73 @@ public CurrentRegions(IImmutableSet<Address> regions)
     /// the state of the shard regions.
     /// </summary>
     [Serializable]
-    public sealed class GetClusterShardingStats : IShardRegionQuery
+    public sealed class GetClusterShardingStats : IShardRegionQuery, IClusterShardingSerializable
     {
         /// <summary>
-        /// TBD
+        /// The timeout for this operation.
         /// </summary>
         public readonly TimeSpan Timeout;
 
         /// <summary>
-        /// TBD
+        /// Creates a new GetClusterShardingStats message instance.
         /// </summary>
-        /// <param name="timeout">TBD</param>
+        /// <param name="timeout">The amount of time to allow this operation to run.</param>
         public GetClusterShardingStats(TimeSpan timeout)
         {
             Timeout = timeout;
         }
+
+        private bool Equals(GetClusterShardingStats other)
```

Comment:
If you created equality rules, please make that type implement IEquatable explicitly.